### PR TITLE
[dist] Add Event stats

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/script/delayed_job_stats.rb [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+event_types = event_classes.flat_map(&:descendants).map(&:name)
+oldest_date = 10.days.ago
+project_logged = Event::Base.where(project_logged: false, eventtype: event_types).where(["created_at > ?", oldest_date]).count
+mails_sent = Event::Base.where(mails_sent: false).count
+undone_jobs = Event::Base.where.not(undone_jobs: 0).count
+puts "There are currently #{project_logged + mails_sent + undone_jobs} Events to be processed"
+puts "---------Events by type-----------"
+puts "ProjectLogs to create".ljust(30) + project_logged
+puts "Mails to send".ljust(30) + mails_sent
+puts "Backend jobs to do".ljust(30) + undone_jobs
```
Comment (eduardoj) [inline src/api/script/delayed_job_stats.rb]: Is `undone_jobs` the number of "undone jobs" or the number of "events with undone jobs"?

Comment (hennevogel) [reply src/api/script/delayed_job_stats.rb]: The number of Events that have undone jobs...

Comment (eduardoj) [reply src/api/script/delayed_job_stats.rb]: I would change the string to "Events with backend jobs to do". What you prefer.

Comment (hennevogel) [reply src/api/script/delayed_job_stats.rb]: The headline says its about Events right?. Look at the output

```
-----------EVENT-STATS------------
There are currently 35396 Events to be processed
---------Events by type-----------
Project RSS Feed              35327
Mails to send                 69
Backend jobs to do            93356
-------------DJ-STATS-------------
There are currently 4839 jobs in the queues
-----------Jobs by type-----------
EventNotifyBackendJob          1
PackageUpdateIfDirtyJob        2352
ProjectLogRotateJob            2485
SendEventEmailsJob             1
-----------Jobs by queue----------
quick                          1
mailers                        1
default                        2352
project_log_rotate             2485
```

Comment (eduardoj) [reply src/api/script/delayed_job_stats.rb]: > The headline says its about Events right? Look at the output:

Fine. Fair enough.

**Pull request merged**